spec/std/range_spec.cr

```diff
@@ -385,6 +385,40 @@ describe "Range" do
     end
   end
 
+  describe "sample" do
+    it "raises on open range" do
+      expect_raises(ArgumentError, "Can't sample an open range") do
+        (1..(true ? nil : 1)).sample
+      end
+      expect_raises(ArgumentError, "Can't sample an open range") do
+        ((true ? nil : 1)..1).sample
+      end
+      expect_raises(ArgumentError, "Can't sample an open range") do
+        ((true ? nil : 1)..(true ? nil : 1)).sample
+      end
+    end
+
+    it "samples a float range as a distribution" do
+      r = (1.2..3.4)
+      x = r.sample
+      r.includes?(x).should be_true
```

Comment:
@straight-shoota This is resolved already??

Reply:
It's not. @straight-shoota wants it to be like this:

```
x.in?(r).should be_true
```

In my mind these are almost the same because there's no specific matcher for this assertion. So I think this can be merged, but let's wait for @straight-shoota 's suggestion.

Reply:
Oh, thank you. Sorry, I somehow reviewed the wrong merge commit and assumed something was broken.

Reply:
@asterite No this is fine. I just didn't want `r.sample` (i.e. `x`) to be on the right hand side in the expectation. `x.in?(r)` vs. `r.includes?(x)` doesn't matter.
I'd still prefer `x.should be_in(r)`, though to make this even better :P

Reply:
But it's still on the right hand side...?

Reply:
No? Not sure which specific you refer to, but neither `be_true` nor `be_in(r)` have `r.sample`/`x` in them.